assets/sass/config/_variables.scss

```diff
@@ -64,10 +64,12 @@ $c-mango-tango: #e77d5b;
 $c-thunderbird: #ac4220;
 $c-solitude: #f8faff;
 $c-heather: #b8bdb9;
-$c-gallery: #ebeef0;
-$c-gallery-2: #ebeef0;
+$c-gallery: #ededed;
+$c-porcelain: #ebeef0;
+$c-pumice: #c4c7c5;
+$c-vivid-blue: #1a73e8;
 $c-fiord: #1b3870;
-$c-iron: #b8bdb9;
+$c-iron: #dadce0;
```

Comment:
Some of these color names were out of sync with http://chir.ag/projects/name-that-color/.